test/acceptance/features/unbindAppToService.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Unbind an application from a service
+
+    As a user of Service Binding Operator
+    I want to unbind applications from services
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+        * PostgreSQL DB operator is installed
+
+    @disabled
+    Scenario: Unbind a generic test application to PostgreSQL database
```

Comment:
Nit - :see_no_evil:  unbind `from` the service

Reply:
Done.